compiler/aarch64/codegen/FPTreeEvaluator.cpp

```diff
@@ -196,17 +196,24 @@ OMR::ARM64::TreeEvaluator::fconstEvaluator(TR::Node *node, TR::CodeGenerator *cg
       }
    else
       {
-      TR::Register *tmpReg = cg->allocateRegister();
-
       union {
          float f;
          int32_t i;
       } fvalue;
 
       fvalue.f = node->getFloat();
-      loadConstant32(cg, node, fvalue.i, tmpReg);
-      generateTrg1Src1Instruction(cg, TR::InstOpCode::fmov_wtos, node, trgReg, tmpReg);
-      cg->stopUsingRegister(tmpReg);
+
+      if(fvalue.f == 0.0f)
```

Comment:
The condition is true when `fvalue.f` is -0.0f.
You cannot use the `movi0s` opcode for loading -0.0f to a FP register.
Please consider using `signbit()` function for checking -0.0f.

Reply:
Hi Konno-san, I have updated the code & would appreciate a review. Thank you.